### ISSUE TITLE
Error with new `reportlab` dependency version 3.6.7 

### ISSUE THREAD

**Issue by xdekasx** [xhtml2pdf/xhtml2pdf#588]
I realize that the dependency `reportlab` has updated to 3.6.7 version, which removes `getStringIO` used in this package.
After this, raise an error.

Here the log.

```
File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/pisa.py", line 19, in <module>
    from xhtml2pdf.document import pisaDocument
  File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/document.py", line 4, in <module>
    from xhtml2pdf.context import pisaContext
  File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/context.py", line 11, in <module>
    import xhtml2pdf.parser
  File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/parser.py", line 32, in <module>
    from xhtml2pdf.tables import *  # TODO: Kill wild import!
  File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/tables.py", line 8, in <module>
    from xhtml2pdf.tags import pisaTag
  File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/tags.py", line 10, in <module>
    from xhtml2pdf import xhtml2pdf_reportlab
  File "/usr/local/lib/python3.8/site-packages/xhtml2pdf/xhtml2pdf_reportlab.py", line 20, in <module>
    from reportlab.lib.utils import flatten, open_for_read, getStringIO, \
ImportError: cannot import name 'getStringIO' from 'reportlab.lib.utils' (/usr/local/lib/python3.8/site-packages/reportlab/lib/utils.py)
```

**Comment by despatates** [comment]
Solution is to freeze version to 3.6.6. See https://github.com/xhtml2pdf/xhtml2pdf/issues/589.

**Comment by luisza** [comment]
Fixed on #590
